modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/data_set/filters/modals/selection_filter/source_type/Picklist.tsx

```diff
@@ -13,23 +13,21 @@ import ValidationFeedback from '../../../../../components/ValidationFeedback';
 import getAllPicklists from '../../../../../utils/getAllPicklists';
 import {IFilter, IPickList} from '../../../../../utils/types';
 
-interface IObjectPicklistProps {
-	filter?: IFilter;
-	namespace: string;
-	onChange: Function;
-	sourceValidationError: boolean;
-}
-
-function ObjectPicklist({
+function Picklist({
```

Comment:
I renamed `ObjectPicklist*` to `Picklist*` for consistency, we were already calling this `Picklist` in many places.